.prettierrc

```diff
@@ -0,0 +1,22 @@
+{
+"printWidth":140,
+"tabWidth":4,
+"useTabs":true,
+"trailingComma":"none",
+"overrides": [
+        {
+            "files": "*.{pcss,scss,css,json}",
+            "options": {
+                "tabWidth": 2,
+                "useTabs":false
+            }
+        },
+        {
+        	"files": "{package.json,.travis.yml}",
```

Comment:
Since json files are taken in "files": "*.{pcss,scss,css,**json**}", package.json should not be mentionned here. Could you please remove it ? :)